### PR TITLE
Feature/wk 186 store documents

### PR DESCRIPTION
**Pull request by jklardie**

### CODE REVIEW

File: api/storage/objectstore.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+XLS_OBJECT_NAME = 'VVP_Blackspot_Voortgangslijst_Kaart_actueel.xls'
+DOWNLOAD_DIR = '/tmp/blackspots/'
+WBA_CONTAINER_NAME = 'wbalijst'
+
+DocumentList = List[Tuple[str, str]]
+
+
+class ObjectStore:
+
+    logger = logging.getLogger(__name__)
```
Comment (kramer65) [inline api/storage/objectstore.py]: I would place the logger outside the class

File: api/storage/objectstore.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        output_path = os.path.join(DOWNLOAD_DIR, object_name)
+
+        if os.path.isfile(output_path):
+            self.logger.info(f"Using cached file: {object_name}")
+        else:
+            self.logger.info(f"Fetching file: {object_name}")
+            new_data = connection.get_object(container_name, object_name)[1]
+            with open(output_path, 'wb') as file:
+                file.write(new_data)
+        return output_path
```
Comment (kramer65) [inline api/storage/objectstore.py]: The files could change outside the scope of this application. Caching is therefore a dangerous thing to do. I would remove the caching mechanism.

File: api/api/views.py [hunk 3]
```diff
@@ -2,7 +2,9 @@
 from datetime import date
 
 from datapunt_api.rest import DatapuntViewSet
-from django.http import Http404, HttpResponse, HttpResponseServerError, StreamingHttpResponse
+from django.conf import settings
+from django.http import Http404, HttpResponse, HttpResponseServerError
+from django.http import StreamingHttpResponse
```
Comment (kramer65) [inline api/api/views.py]: Is importing these on separate lines done because of flake8?

File: api/storage/objectstore.py (hunk 2 above)
Comment (jklardie) [reply api/storage/objectstore.py]: Agreed, good catch. This code will be removed once this PR has proven to be successful.

File: api/api/views.py (hunk 3 above)
Comment (jklardie) [reply api/api/views.py]: Nope. Flake is quite mad about it actually. Fixing now.

**Pull request merged**